tests/integration/test_exec.py

```diff
@@ -1,12 +1,35 @@
 __copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
-from jina import Flow, Document
-from jinahub.SUB_PACKAGE_NAME.MODULE_NAME import MyDummyExecutor
+import os
+import librosa
 
+from jina import Flow, Document, DocumentArray
+from vggish import vggish_input
 
-def test_exec():
-    f = Flow().add(uses=MyDummyExecutor)
-    with f:
-        resp = f.post(on='/test', inputs=Document(), return_results=True)
-        assert resp is not None
+try:
+    from vggish_audio_encoder import VggishAudioEncoder
+except:
+    from jinahub.encoders.audio.vggish_audio_encoder import VggishAudioEncoder
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def test_flow_from_yml():
+
+    doc = DocumentArray([Document()])
+    with Flow.load_config(os.path.join(cur_dir, 'flow.yml')) as f:
+        resp = f.post(on='test', inputs=doc, return_results=True)
+
+    assert resp is not None
+
```

Comment:
more tests such as size of the output embedding?

Reply:
Done as unit test